portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,30 +35,41 @@
   @Override
   public void init() {
     messages = new ArrayList<>();
-    messages.add("Once upon a time...");
-    messages.add("Twice upon two times...");
-    messages.add("Thrice upon three times...");
   }
 
+  /** Gets the messages stored in the messages variable in JSON format */
```

Comment:
Perhaps in this comment it would be worth noting that these messages are form the user's form input on the HTML page.

Reply:
Done!